### PR TITLE
[docs]Update ascend_quick_start doc

### PR DESCRIPTION
**Pull request by Alter-A1ways**
### What does this PR do?

Update ascend_quick_start doc, and provide a example yaml for qwen3_32b sft

> Add **concise** overview of what this PR aims to achieve or accomplish. Reference related GitHub issues and PRs that help with the review.

### Checklist Before Starting

- [x] Search for similar PRs. Paste at least one query link here: ...
- [x] Format the PR title as `[{modules}] {type}: {description}` (This will be checked by the CI)
  - `{modules}` include `misc`, `ci`, `config`, `docs`, `core`, `data`, `dist`, `omni`, `logging`, `model`, `optim`, `ckpt`, `release`, `task`, `
  - If this PR involves multiple modules, separate them with `,` like `[megatron, torch, liger-kernals,fsdp, doc]`
  - `{type}` is in `feat`, `fix`, `refactor`, `chore`, `test`
  - If this PR breaks any API (CLI arguments, config, function signature, etc.), add `[BREAKING]` to the beginning of the title.
  - Example: `[BREAKING][fsdp, torch, liger-kernals] feat: dynamic batching`

### Test

> For changes that can not be tested by CI (e.g., algorithm implementation, new model support), validate by experiment(s) and show results like training curve plots, evaluation results, etc.

### API and Usage Example

> Demonstrate how the API changes if any, and provide usage example(s) if possible.

```python
# Add code snippet or script demonstrating how to use this
```

### Design & Code Changes

> Demonstrate the high-level design if this PR is complex, and list the specific changes.

### Checklist Before Submitting

> [!IMPORTANT]
> Please check all the following items before requesting a review, otherwise the reviewer might deprioritize this PR for review.

- [x] Read the [Contribute Guide](https://github.com/ByteDance-Seed/VeOmni/blob/main/CONTRIBUTING.md).
- [x] Apply [pre-commit checks](https://github.com/ByteDance-Seed/VeOmni/blob/main/CONTRIBUTING.md?plain=1#L22): `make commit && make style && make quality`
- [ ] Add / Update [the documentation](https://github.com/ByteDance-Seed/VeOmni/blob/main/docs).
- [ ] Add unit or end-to-end test(s) to [the CI workflow](https://github.com/ByteDance-Seed/VeOmni/tree/main/.github/workflows) to cover all the code. If not feasible, explain why: ...

### CODE REVIEW

File: docs/ascend_tutorials/ascend_quick_start.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+1. 准备模型和数据集
+
+2. 设置`train.sh`中`NPROC_PER_NODE`参数为实际卡数
+
+3. 运行训练脚本
+
+.. code-block:: bash
+
+    bash train.sh tasks/train_torch.py configs/sft/qwen3_sft.yaml
```
Comment (ji-huazhong) [inline docs/ascend_tutorials/ascend_quick_start.md]:
```suggestion
    PYTORCH_NPU_ALLOC_CONF=expandable_segments:True MULTI STREAM MEMORY REUSE=2 bash train.sh tasks/train_torch.py configs/sft/qwen3_sft.yaml
```

For optimal performance, it is recommended to set `export MULTI STREAM MEMORY REUSE=2` and `export PYTORCH_NPU_ALLOC_CONF=expandable_segments:True` on the A3.

File: docs/ascend_tutorials/ascend_quick_start.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+基础环境准备请参照这份 `文档 <https://gitcode.com/Ascend/pytorch>`_ 。
+
+### 安装VeOmni
+
+.. code-block:: bash
+
+    git clone https://github.com/ByteDance-Seed/VeOmni.git
+    cd VeOmni
+    pip install -e .[npu]
```
Comment (pjgao) [inline docs/ascend_tutorials/ascend_quick_start.md]: 建议使用uv，不要用pip

File: docs/ascend_tutorials/ascend_quick_start.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+1. 准备模型和数据集
+
+2. 设置`train.sh`中`NPROC_PER_NODE`参数为实际卡数
+
+3. 运行训练脚本
+
+.. code-block:: bash
+    # 设置环境变量
+    export PYTORCH_NPU_ALLOC_CONF=expandable_segments:True 
+    export MULTI_STREAM_MEMORY_REUSE=2
```
Comment (pjgao) [inline docs/ascend_tutorials/ascend_quick_start.md]: 多流复用Q3商发是不是还没上？这里会不会有精度风险？

File: docs/ascend_tutorials/ascend_quick_start.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
+# Ascend Quickstart
+
+Last updated: 11/27/2025
+
+我们在VeOmni上增加对华为昇腾设备的支持。
+
+### 基础环境准备
+
++-----------+--------------------------+
```
Comment (Crystal-jiang) [inline docs/ascend_tutorials/ascend_quick_start.md]: markdown表格格式存在问题

File: docs/ascend_tutorials/ascend_quick_start.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+.. code-block:: bash
+    # 设置环境变量
+    export PYTORCH_NPU_ALLOC_CONF=expandable_segments:True 
+    export MULTI_STREAM_MEMORY_REUSE=2 
+    bash train.sh tasks/train_torch.py configs/sft/qwen3_sft.yaml
+
+### 并行能力支持
+
++-----------+--------------------------+
```
Comment (Crystal-jiang) [inline docs/ascend_tutorials/ascend_quick_start.md]: markdown表格格式存在问题

File: docs/ascend_tutorials/ascend_quick_start.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+### 快速开始
+
+1. 准备模型和数据集
+
+2. 设置`train.sh`中`NPROC_PER_NODE`参数为实际卡数
+
+3. 运行训练脚本
+
+.. code-block:: bash
```
Comment (Crystal-jiang) [inline docs/ascend_tutorials/ascend_quick_start.md]: markdown bash代码格式存在问题

File: docs/ascend_tutorials/ascend_quick_start.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+| torch     | == 2.7.1                 |
++-----------+--------------------------+
+| torch_npu | == 2.7.1                 |
++-----------+--------------------------+
+
+基础环境准备请参照这份 `文档 <https://gitcode.com/Ascend/pytorch>`_ 。
+
+### 安装VeOmni
+
+.. code-block:: bash
```
Comment (Crystal-jiang) [inline docs/ascend_tutorials/ascend_quick_start.md]: markdown bash代码格式存在问题

File: docs/ascend_tutorials/ascend_quick_start.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
+# Ascend Quickstart
+
+Last updated: 11/28/2025
+
+我们在VeOmni上增加对华为昇腾设备的支持。
```
Comment (onehaitao) [inline docs/ascend_tutorials/ascend_quick_start.md:5]: 建议同意使用英文或者有两个版本（纯中文版本/纯英文版本）

File: docs/ascend_tutorials/ascend_quick_start.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+| torch_npu | == 2.7.1                 |
+
+基础环境准备请参照这份 [文档](https://gitcode.com/Ascend/pytorch) 。
+
+### 安装VeOmni
+
+```bash
+git clone https://github.com/ByteDance-Seed/VeOmni.git
+cd VeOmni
+pip install -e .[npu]
```
Comment (onehaitao) [inline docs/ascend_tutorials/ascend_quick_start.md]: 建议使用uv sync的方式安装环境

**Comment by onehaitao** [comment]
label ascend
